impl/src/main/java/org/ehcache/impl/internal/statistics/DefaultStatisticsServiceFactory.java

```diff
@@ -18,13 +18,17 @@
 
 import org.ehcache.core.spi.service.ServiceFactory;
 import org.ehcache.core.spi.service.StatisticsService;
+import org.ehcache.core.spi.service.StatisticsServiceConfiguration;
 import org.ehcache.spi.service.ServiceCreationConfiguration;
 
 public class DefaultStatisticsServiceFactory implements ServiceFactory<StatisticsService> {
 
   @Override
   public StatisticsService create(ServiceCreationConfiguration<StatisticsService> serviceConfiguration) {
-    return new DefaultStatisticsService();
+    StatisticsServiceConfiguration configuration = serviceConfiguration != null && serviceConfiguration instanceof StatisticsServiceConfiguration ?
```

Comment:
This is weird. If it doesn't instance of, we should throw an exception I think. Silently returning an empty configuration seems sneaky.

Reply:
Please test the behavior

Reply:
no this is what all Ehcache services do, there's a default, otherwise the user is forced to pass a configuration to use the service, and we have to annotate with `@ServiceFactory.RequiresConfiguration`. This use case is like the management config which can be optionally configured.